packages/calendar/src/Calendar.ts

```diff
@@ -286,36 +304,29 @@ export class Calendar extends SpectrumElement {
         `;
     }
 
-    protected renderCalendarHeader(): TemplateResult {
-        const monthAndYear = this.formatDate(this.currentDate, {
+    private get monthAndYear(): string {
+        return this.formatDate(this.currentDate, {
             month: 'long',
             year: 'numeric',
         });
+    }
 
+    protected renderCalendarHeader(): TemplateResult {
         return html`
             <div class="header" @focusin=${this.resetDateFocusIntent}>
-                <!--
-                 * TODO: Attribute 'role="heading"' removed, due to error 'The "heading" role requires the attribute
-                 * "aria-level"'
-                -->
-                <div
+                <h2
                     class="title"
                     aria-live="assertive"
```

Comment:
This seems to not work on Safari with VoiceOver, it always announces the initial month-year of the Calendar. 
I think a general approach similar to the [React Spectrum's Live Announcer](https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/live-announcer/src/LiveAnnouncer.tsx) might fix this issue.

Reply:
I think we would want to use `polite`, not `assertive`, so that if any other things are read, the screen reader would wait for those to finish first before reading this one